src/Host.h

```diff
@@ -444,15 +444,15 @@ class Host : public QObject
     bool mEchoLuaErrors;
     QFont mCommandLineFont;
     QString mCommandSeparator;
-    bool mEnableGMCP;
-    bool mEnableMSSP;
-    bool mEnableMSDP;
-    bool mEnableMSP;
+    bool mEnableGMCP = true;
+    bool mEnableMSSP = true;
+    bool mEnableMSDP = true;
```

Comment:
Researching the why on this one.

Reply:
Um, I was just reproducing what was done before:
![image](https://github.com/user-attachments/assets/2510a666-57d2-4db1-9c12-defe5c3ba092)

The `mEnableGMCP` one can be traced back to #386, the `mEnableMSSP` one to #3025, however the `mEnableMSDP` is my fault and down to #7224 as you report...